### DISCUSSION TITLE
A call to return sales history

### DISCUSSION THREAD

**Discussion by j-algo** [amzn/selling-partner-api-models#3872] [General]
Is there a way to get the past/recent history of sales of a listing? 
Eg. given the ASIN, marketplace-ID and a number of days, the call will return how any of that ASIN has been sold within the last that-many days in that marketplace. 

Is this possible? In which model?

I'm using the Java SP-API. 

I haven't been able to see anything, but don't want to miss out. Please also verify if no such feature. 

TIA.

**Comment by schwaaamp** [comment]
This appears to be possible using the Sales model: https://developer-docs.amazon.com/sp-api/docs/sales-api-v1-reference
